openshift-kube-apiserver/admission/autoscaling/apis/runonceduration/validation/validation.go

```diff
@@ -1,8 +1,8 @@
 package validation
 
 import (
-	"k8s.io/kubernetes/openshift-kube-apiserver/admission/autoscaling/apis/runonceduration"
```

Comment:
no generated

Reply:
Added new commit `UPSTREAM: <drop>: Update gofmt` that makes these changes.